flink-queryable-state/flink-queryable-state-runtime/src/test/java/org/apache/flink/queryablestate/itcases/HAQueryableStateFsBackendITCase.java

```diff
@@ -96,7 +96,7 @@ private static Configuration getConfig() throws Exception {
 
 		Configuration config = new Configuration();
 		config.setBoolean(QueryableStateOptions.ENABLE_QUERYABLE_STATE_PROXY_SERVER, true);
-		config.setString(TaskManagerOptions.MANAGED_MEMORY_SIZE, "4m");
+		config.setString(TaskManagerOptions.LEGACY_MANAGED_MEMORY_SIZE, "4m");
```

Comment:
Should we replace these config keys with the non deprecated version?

Reply:
Not until we remove the feature option and clean up the legacy codes.
Currently, FLIP-49 is disabled by default. If we use the new key here, it will be ignored by the legacy code path.

Reply:
ok, makes sense.